xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsApplicationsPreferencesMacro.xml

```diff
@@ -1044,7 +1044,12 @@ require(['jquery', 'xwiki-meta', 'ApplicationWidget', 'xwiki-bootstrap-switch',
       #end
       ## Collapse button
       &lt;td class="notificationCollapseButtonCell"&gt;
-        &lt;button type="button" class="btn btn-default btn-sm collapseButton"&gt;$services.icon.renderHTML('caret-down')&lt;/button&gt;
+        &lt;button type="button" class="btn btn-default btn-sm collapseButton"&gt;
+          $services.icon.renderHTML('caret-down')
+          &lt;label class='sr-only'&gt;
+            $escapetool.xml($services.localization.render('notifications.settings.applications.table.dropdown.toggle.label',[$type.applicationName]))
+          &lt;/label&gt;
```

Comment:
The use of `<label>` is not allowed here, button elements don't allow interactive content and `<label>` is interactive content.

Reply:
Source: https://html.spec.whatwg.org/multipage/dom.html#interactive-content